diff --git a/node_modules/@lerna/version/index.js b/node_modules/@lerna/version/index.js
index 3fcea7c..8c891be 100644
--- a/node_modules/@lerna/version/index.js
+++ b/node_modules/@lerna/version/index.js
@@ -98,7 +98,7 @@ class VersionCommand extends Command {
     };
 
     // https://docs.npmjs.com/misc/config#save-prefix
-    this.savePrefix = this.options.exact ? "" : "^";
+    this.savePrefix = this.options.exact ? "" : "~";
   }
 
   initialize() {
