diff --git a/node_modules/lerna/dist/index.js b/node_modules/lerna/dist/index.js
index 0860a73..7fae274 100644
--- a/node_modules/lerna/dist/index.js
+++ b/node_modules/lerna/dist/index.js
@@ -9914,7 +9914,7 @@ var require_src2 = __commonJS({
           forceGitTag,
           overrideMessage
         };
-        this.savePrefix = this.options.exact ? "" : "^";
+        this.savePrefix = this.options.exact ? "" : "~";
       }
       async initialize() {
         if (!this.project.isIndependent()) {
